.github/workflows/rust.yml

```diff
@@ -130,6 +130,10 @@ jobs:
       env:
         LIBSQL_EXPERIMENTAL_PAGER: 1000
       run: cargo nextest run
+
+    - name: Run tests with durable WAL
```

Comment:
Is that necessary, since even with the feature on, tests only run with vanilla wal, if I understand correctly?

Reply:
if so, no need to lengthen the CI for nothing

Reply:
I wanted to use durable wal in the runner, but I forgot to remove it. I have removed it now